index.js

```diff
@@ -10,13 +10,13 @@ const {gitHead: getGitHead, isGitRepo} = require('./lib/git');
 module.exports = async opts => {
   const {isCi, branch, isPr} = envCi();
 
-  if (!isCi && !opts.dryRun) {
+  if (!isCi && !opts.dryRun && opts.noCi !== true) {
```

Comment:
why ` !== true` instead of `!opts.noCi`?

Reply:
No reason, it was a mistake. I changed it.